cosipy/image_deconvolution/RichardsonLucy.py

```diff
@@ -66,16 +70,26 @@ def __init__(self, initial_model, dataset, mask, parameter):
             else:
                 os.makedirs(self.save_results_directory)
 
```

Comment:
I understand that RL needs to know if it is performed on the master node and needs this kind of parameter. I would suggest preparing two parameters alternatively, something like

- self.parallel_computation = True / False
- self.master_node = True / False

I want to prepare a parameter that explicitly tells if the computation is in parallel or not. I will add some suggestions regarding these changes at other lines.